src/lazy/sequence.rs

```diff
@@ -94,24 +96,18 @@ impl<'top, 'data> LazySequence<'top, 'data> {
     ///# Ok(())
     ///# }
     /// ```
-    pub fn annotations(&self) -> AnnotationsIterator<'top, 'data> {
+    pub fn annotations(&self) -> AnnotationsIterator<'top, 'data, F> {
         AnnotationsIterator {
-            raw_annotations: self.raw_sequence.value.annotations(),
+            raw_annotations: self.raw_sequence.as_value().annotations(),
             symbol_table: self.symbol_table,
-            initial_offset: self
```

Comment:
🗺️ The `initial_offset` member field was a copy/pasted holdover from the raw implementation. I've removed it in each of the user-level types that had erroneously retained it.